src/test/java/VehicleTest.java

```diff
@@ -98,16 +84,6 @@ public void testVehicleCapacity1() {
 	}
 	
 	@Test
-	public void testVehicleCapacity2() {
-		try {
-			Vehicle v = new Vehicle("Car", 10, 10, -1, 0, 0, 0);
-			fail("No exception thrown.");
```

Comment:
these tests all seemed redundant but it's tough to know. you still have 100% code coverage with all these removed so it's proably fine (probably?)